7_react/react2/week1/theme/src/components/ThemedHeader.js

```diff
@@ -0,0 +1,8 @@
+import { useContext } from "react";
+import { ThemeContext } from "../contexts/ThemeContext";
+
+export default function ThemedHeader({ title }) {
+  const theme = useContext(ThemeContext);
+  const classes = `themed-header ` + theme;
+  return <h2 className={classes}>{title}</h2>;
+}
```

Comment:
Here we read out the value of the ThemeContext in one of our components. Notice how we don't have to pass it via props.